Core/include/Acts/Definitions/PdgParticle.hpp

```diff
@@ -25,6 +25,8 @@ enum PdgParticle : std::int32_t {
   eGamma = 22,
   ePionZero = 111,
   ePionPlus = 211,
+  eKaon = 321,
+  eAntiKaon = -eKaon,
   ePionMinus = -ePionPlus,
```

Comment:
```suggestion
  ePionMinus = -ePionPlus,
  eKaonPlus = 321,
  eKaonMinus = -eKaon,
```

looking at the pions this would be more consistent